plugin/restore/mysql/mysql.go

```diff
@@ -664,6 +664,18 @@ func checkVersionForPITR(version string) error {
 	return nil
 }
```

Comment:
Let's switch GetVersion() to getServerVariable().

Reply:
This needs a refactor, otherwise will introduce circular dependencies.
I will do it in another PR.